Sources/Filters/General/ImageDataOutlineFilter/index.js

```diff
@@ -0,0 +1,149 @@
+import { vec3 } from 'gl-matrix';
+
+import macro from 'vtk.js/Sources/macros';
+import vtkCubeSource from 'vtk.js/Sources/Filters/Sources/CubeSource';
+import vtkPolyData from 'vtk.js/Sources/Common/DataModel/PolyData';
+import vtkCellArray from '../../../Common/Core/CellArray';
+
+const { vtkErrorMacro } = macro;
+
+// prettier-ignore
+export const LINE_ARRAY = [
+  2, 0, 1,
+  2, 2, 3,
+  2, 4, 5,
+  2, 6, 7,
+  2, 0, 2,
+  2, 1, 3,
+  2, 4, 6,
+  2, 5, 7,
+  2, 0, 4,
+  2, 1, 5,
+  2, 2, 6,
+  2, 3, 7,
+];
+
+// ----------------------------------------------------------------------------
+// vtkImageDataOutlineFilter methods
+// ----------------------------------------------------------------------------
+
+function vtkImageDataOutlineFilter(publicAPI, model) {
+  // Set our className
+  model.classHierarchy.push('vtkImageDataOutlineFilter');
+
+  function transformPoints(points, transform) {
+    const tmp = [0, 0, 0];
+    for (let i = 0; i < points.length; i += 3) {
+      const p = points.subarray(i, i + 3);
```

Comment:
`subarray` is typically a slow operation. 

It's faster to copy points manually

You may also use `vtkTransform.transformPoints` (to improve to support Mat4) or `vtkMatrixBuilder.apply`

Reply:
Yes. Changed to use `vtkTransform.transformPoints`.